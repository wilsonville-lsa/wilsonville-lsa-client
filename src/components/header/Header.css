h1, h2 {
  color: rgb(95, 87, 87);
}

nav {
  border: 5px outset rgb(79, 210, 233);
  background-color: rgb(205, 234, 235);
  width: 70vw;
  margin: auto;
}

nav a {
  margin: 2vh 2vw;
}

nav a:link, nav a:visited {
  text-decoration: none;
  color: green;
}

.active, nav a:hover {
  color: rgb(9, 177, 9);
}