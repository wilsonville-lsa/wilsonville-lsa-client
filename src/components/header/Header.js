/* eslint-disable react/no-unescaped-entities */
import React from 'react';
import { Link } from 'react-router-dom';

export default function Header() {
  return (
    <>
      <h1>Wilsonville Baha'i Community</h1>
      <h2>BAHA'I FAITH</h2>

      <nav>
        <Link to="/">Home</Link>
      </nav>
    </>
  );
}
