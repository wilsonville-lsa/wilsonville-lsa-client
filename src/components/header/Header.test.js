import React from 'react';
import renderer from 'react-test-renderer';
import { MemoryRouter } from 'react-router-dom';
import Header from './Header';

describe('Header', () => {
  it('matches snapshot for Header', () => {
    const tree = renderer.create(
      <MemoryRouter>
        <Header />
      </MemoryRouter>
    );

    expect(tree).toMatchSnapshot();
  });
});
