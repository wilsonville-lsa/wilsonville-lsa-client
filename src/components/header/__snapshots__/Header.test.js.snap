// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header matches snapshot for Header 1`] = `
Array [
  <h1>
    Wilsonville Baha'i Community
  </h1>,
  <h2>
    BAHA'I FAITH
  </h2>,
  <nav>
    <a
      href="/"
      onClick={[Function]}
    >
      Home
    </a>
  </nav>,
]
`;
